data-access/hdp_data_access.ipynb

```diff
@@ -0,0 +1,236 @@
+    "Data access for our historical weather station data is still a work in progress as we build a data catalog and continue generating data products. Eventually, helper functions will be incorporated into `climakitae` to streamline data access. The code repository for the data generation, including a full QAQC protocol, will be made publically available on GitHub by the end of summer 2025. \n",
+    "For more information on this data, and the associated project at Eagle Rock Analytics (Historical Observations Data Platform), see the following page: [https://eaglerockanalytics.com/project/historical-observations-data-platform/](https://eaglerockanalytics.com/project/historical-observations-data-platform/)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ccc34b3e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import intake "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d9d4eb1c",
+   "metadata": {},
+   "source": [
+    "First, open the catalog using `intake`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6ccad77a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cat = intake.open_esm_datastore(\"https://cadcat.s3.amazonaws.com/histwxstns/era-hdp-collection.json\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "989e64b5",
+   "metadata": {},
+   "source": [
+    "Next, view the catalog in table format. You can inspect the first few rows by calling `.head()` on the table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0389d0d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Access catalog as dataframe and inspect the first few rows\n",
+    "cat_df = cat.df\n",
+    "cat_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e57cfbd5",
+   "metadata": {},
+   "source": [
+    "View all the weather station networks by using the following code "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c743e382",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# See all network options \n",
+    "cat_df[\"network_id\"].unique()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4864dd8c",
+   "metadata": {},
+   "source": [
+    "You can also filter the catalog to see all stations within a network"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "687f661b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_network = \"ASOSAWOS\"\n",
+    "cat_df[cat_df[\"network_id\"] == my_network]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3ddfe314",
+   "metadata": {},
+   "source": [
+    "You can subset the catalog and read in the Zarrs as `xarray.Dataset` objects using the method shown below. To change the data downloaded, simply modify the inputs in the dictionary `query`. These inputs must correspond to valid options in the catalog. "
```

Comment:
Not sure how much the users know about Zarr but this looks like the first time it gets mentioned. If we're referencing the file type could be nice to state more explicitly that the data is stored in Zarr format but we can load it into xarray.

Reply:
Replaced "Zarr" with "cloud-optimized data"